style.css

```diff
@@ -0,0 +1,815 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --secondary-color: #ec5242;
+  --gray-color: #d3d3d3;
+  --white-color: #fff;
+  --primary-color: #272a31;
+  --gray-color-2: #f3f3f3;
+  --paragraph-color: #272a31;
+  --soft-black: #3c3f46;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
+  color: var(--primary-color);
+  background-color: #fff;
+  overflow-x: hidden;
+  width: 100%;
+  scroll-behavior: smooth;
+}
+
+/* general css */
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+img {
+  max-width: 100%;
+}
+
+.btn {
+  display: inline-block;
+  padding: 6px 15px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 600;
+  color: var(--white-color);
+  background-color: var(--secondary-color);
+  border: 1px solid var(--secondary-color);
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
+}
+
+.btn:hover {
+  background-color: transparent;
+}
+
+/* top navbar */
+.top-header {
+  background-color: var(--primary-color);
+  color: white;
+  padding: 0.1rem 0;
+  position: relative;
+  overflow: hidden;
+}
+
+.top-header-content {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 1rem;
+  font-size: 12px;
+}
+
+.top-header-content li {
+  transition: all 0.2s ease-in-out;
+}
+
+.navbar-nav li {
+  transition: color 0.2s ease-in-out;
+  font-size: 26px;
+  font-weight: 600;
+}
+
+.navbar-nav li:hover {
+  border-bottom: 2px solid var(--secondary-color);
+  color: var(--gray-color);
+}
+
+.top-header-content li:hover {
+  color: var(--secondary-color);
+}
+
+.no-scroll {
+  overflow-y: hidden;
+}
+
+/* ================media querry for top bar and header */
+@media only screen and (max-width: 768px) {
+  .navbar-header a {
+    /* font-weight: 900; */
+    display: none;
```

Comment:
[Optional] I recommend that you avoid leaving out commented pieces of code in order to have a neater code base. The commit history can be useful to go back in time.